app/controllers/hyrax/admin/workflows_controller.rb

```diff
@@ -0,0 +1,27 @@
+module Hyrax
+  # Presents a list of works in workflow
+  class Admin::WorkflowsController < ApplicationController
+    before_action :ensure_authorized!
+    with_themed_layout 'dashboard'
+    class_attribute :deposited_workflow_state_name
+
+    # Works that are in this workflow state (see workflow json template) are excluded from the
+    # status list and display in the "Published" tab
+    self.deposited_workflow_state_name = 'deposited'
+
+    def index
+      add_breadcrumb t(:'hyrax.controls.home'), root_path
+      add_breadcrumb t(:'hyrax.dashboard.breadcrumbs.admin'), hyrax.dashboard_path
+      add_breadcrumb t(:'hyrax.admin.sidebar.tasks'), '#'
+      add_breadcrumb t(:'hyrax.admin.sidebar.workflow_review'), request.path
+      @status_list = Hyrax::Workflow::StatusListService.new(self, "-workflow_state_name_ssim:#{deposited_workflow_state_name} AND createdby_tesim:#{Contribution::SELFDEP}")
+      @published_list = Hyrax::Workflow::StatusListService.new(self, "workflow_state_name_ssim:#{deposited_workflow_state_name} AND createdby_tesim:#{Contribution::SELFDEP}")
+    end
```

Comment:
Do you need to pull all of this in or could you just reopen the class locally and 
```
def index
    super
         @status_list = Hyrax::Workflow::StatusListService.new(self, "-workflow_state_name_ssim:#{deposited_workflow_state_name} AND createdby_tesim:#{Contribution::SELFDEP}")
       @published_list = Hyrax::Workflow::StatusListService.new(self, "workflow_state_name_ssim:#{deposited_workflow_state_name} AND createdby_tesim:#{Contribution::SELFDEP}")
end
```

That would make the tufts specific changes a little more apparent if you could do something like this.

Reply:
It's pretty challenging to reliably pull in engine code without messing up the rails autoloader.

Reply:
Based on or slack conversation about autoloading, feel free to ignore this.  I'm happy to merge once the tests are green.